### PR TITLE
Update README for new i915 params usage

### PR DESCRIPTION
**Pull request by zhtengw**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,8 +13,8 @@ For Arch Linux users, it is available in AUR. [i915-sriov-dkms-git](https://aur.
 
 Tested kernel versions: 
 
-* `pve-kernel-6.1.0-1-pve`~`6.1.10-1-pve` on PVE VM Host
-* `gentoo-sources-6.1.0-gentoo`~`6.2.0-gentoo` on Gentoo VM Guest
+* `pve-kernel-6.2.9-1-pve` on PVE VM Host
```
Comment (pahowes) [inline README.md:16]: Is the 6.1.x PVE kernel no longer supported?

File: README.md [hunk 2]
```diff
@@ -41,15 +41,15 @@ You can create up to 7 VFs on UHD Graphics 770
 4. Move the entire content of the repository to `/usr/src/i915-sriov-dkms-6.1`. The folder name will be the DKMS package name.
 5. Execute command `dkms -i -m i915-sriov-dkms -v 6.1`. `-m` argument denotes the package name, and it should be the same as the folder name which contains the package content. `-v` argument denotes the package version, which we have specified in the `dkms.conf` as `6.1`
 6. The kernel module should begin building.
-7. Once finished, we need to make a few changes to the kernel commandline. `nano /etc/default/grub` and change `GRUB_CMDLINE_LINUX_DEFAULT` to 'intel_iommu=on i915.enable_guc=7`, or add to it if you have other arguments there already.
+7. Once finished, we need to make a few changes to the kernel commandline. `nano /etc/default/grub` and change `GRUB_CMDLINE_LINUX_DEFAULT` to 'intel_iommu=on i915.enable_guc=3 i915.max_vfs=7`, or add to it if you have other arguments there already.
```
Comment (pahowes) [inline README.md:44]: This change should be more prominent. People who are upgrading from a previous release of `i915-sriov-dkms` may not notice this.

**Review by pahowes — COMMENTED** [review]
Since there is a core change (introduction of the `i915.max_vfs` parameter) is it worth bumping the next release to `v6.2` to prevent confusion? It's a breaking change.

File: README.md (hunk 1 above)
Comment (zhtengw) [reply README.md:16]: Nope, it just means not tested on 6.1 PVE kernel.

File: README.md (hunk 2 above)
Comment (zhtengw) [reply README.md:44]: OK, may add a note for updating from previous version on the top of README.

Comment (FallingSnow) [reply README.md:48]: You can also do this via by adding a udev rule to `/etc/udev/rules.d/intel-sriov.rules` if your OS uses udev.
```
ACTION=="add", DRIVER=="i915", ATTR{sriov_numvfs}="7"
```

**Pull request merged**